tests/unit/test_charm.py

```diff
@@ -3,21 +3,243 @@
 #
 # Learn more about testing at: https://juju.is/docs/sdk/testing
 
-import unittest
 
-import ops
-import ops.testing
+"""Charm unit tests."""
+
+# pylint:disable=protected-access
```

Comment:
Do we really need it? I couldn't see and protected access. 
Suggestion: Do not disable errors on the module level because in the future we may need them.